testsuite/features/support/http_client.rb

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+require 'faraday'
+
+# Wrapper class for HTTP client library (Faraday)
+class HttpClient
+  def initialize(host)
+    puts 'Activating HTTP API'
```

Comment:
Maybe it should say `Activating HTTPs API`

Reply:
? there is only one HTTP.

If you mean `HTTP's API', it's incorrect too.

Reply:
I mean that we are establishing a https connection, not a http connection.

Reply:
Then it should be renamed to `HTTPS`.

Reply:
Hmmm, it's the name we chose for this API in our RFC, in Can's presentation, in the interlock... I do not want to innovate here.

Basically, it works both over HTTP **and** HTTPS. We just happen to block the HTTP version for security reason. So it **is** an HTTP API that we use over HTTPS.

Funnily enough, XML-RPC is also HTTP-based :smile_cat:  . So the name is not even discriminating.